### PR TITLE
Fix all pep8 errors in the repository

### PR DESCRIPTION
**Pull request by Sires0**
Only more **a lot of errors but still fits my screen** to go

Ok, fixed the mixed commits issue

I'm finishing to fix all PEP8, then I'm gonna fix the errors caused...

### CODE REVIEW

File: plyer/platforms/linux/keystore.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 try:
     import keyring
-except:
+except Exception:
```
Comment (tshirtman) [inline plyer/platforms/linux/keystore.py]: Ok, not sure if we can do better there, but that would be nice, excepting on `Exception` is equivalent to just using a bare `except`, so it's a bit bad to do that to follow a style guide.

File: plyer/platforms/macosx/libs/osx_paths.py [hunk 2]
```diff
@@ -1,11 +1,15 @@
 import ctypes
 import os
 
+LibraryPath = \
+    "/System/Library/Frameworks/CoreFoundation.framework/\
+    Versions/A/CoreFoundation"
```
Comment (tshirtman) [inline plyer/platforms/macosx/libs/osx_paths.py]: i think this is a breaking change here, the spaces will be part of the string, you could use something like
```
LibraryPath = ''.join(
    "/System/Library/Frameworks/CoreFoundation.framework/",
   "Versions/A/CoreFoundation"
)
```

or even simply
```
LibraryPath = (
    "/System/Library/Frameworks/CoreFoundation.framework/"
    "Versions/A/CoreFoundation"
)
```
(note the lack of comma at the end of the line, so strings are simply concatenated)

**Comment by Sires0** [comment]
Changes made, please remove your "requested changes", I'm near ready to merge rn.
I just don't understand the why of the assert error at appveyor python 3.6

File: plyer/platforms/macosx/libs/osx_paths.py [hunk 3]
```diff
@@ -1,11 +1,14 @@
 import ctypes
 import os
 
+LibraryPath = "/System/Library/Frameworks/CoreFoundation.framework/"
+"Versions/A/CoreFoundation"
```
Comment (tshirtman) [inline plyer/platforms/macosx/libs/osx_paths.py]: Does this work? i'm pretty sure the second line won't be bart of the string.

```
>>> a = "aaa"
>>> "bbb"
'bbb'
>>> print(a)
aaa
```
but you could just add a `\` at the end of the first line.
```
>>> a = "aaa"\
... "bbb"
>>> print(a)
aaabbb
```

Comment (Sires0) [reply plyer/platforms/macosx/libs/osx_paths.py]: ok

**Comment by Sires0** [comment]
I'd say it's ready to merge, appveyor assert errors seems to be internal, since one time it gives on PEP8, then on PY2 but not on PEP8, AppVeyor seems way 2 broken. And it's not even related to my code

**Comment by tshirtman** [comment]
I rebased and merged manually, thanks!

**Pull request closed without merging**